src/styles/sponsors.module.css

```diff
@@ -0,0 +1,120 @@
+.jumbotron {
+    background-image: url(https://images.unsplash.com/photo-1519999482648-25049ddd37b1?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=0471f46a1cda51342fdecf9a8c42349f&auto=format&fit=crop&w=1852&q=80);
+    background-size: cover;
+}
+
+.heading {
+    font-size: 6em;
+    font-weight: 3em;
+    color: #ffffff;
+}
+
+.headingContainer {
+    margin-bottom: 2em;
+    border-bottom: 3px solid #eeeeee;
+}
+
+.blurb {
+    font-size: 2.5em;
+    font-weight: 2em;
+}
+
+.sponsorsLevel {
+    font-weight: bold;
+    font-size: 1.4rem;
+    text-transform: uppercase;
+    color: #000000;
+    text-align: center;
+}
+
+.logostyle {
+    height: 300px;
+    width: 300px;
+
+}
+
+.gallery-block {
+	padding-bottom: 60px;
+    padding-top: 60px;
+}
+
+.rowStyling {
+    justify-content: center;
+}
+
+.sponsorCard {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 1em;
+    margin-bottom: 1em;
+}
+
+.cardImgContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.sponsorName {
+    color: #000000;
+    font-size: 2em;
+}
+
+.cardBody {
+    border-top: 5px solid #000000;   
+    text-align: center;
+}
+
+.footer {
+    border-top: 5px solid #000000;
+
+}
+
+.contactUs{
+    padding-left: 10px;
+}
+
+.footer{
+    display: flex;
+    text-align: center;
+    align-self: center;
+    justify-content: center;
+}
+
+.footerBlurb {
+    text-align: center;
+    font-size: 1.8em;
+    font-weight: 2em;
+}
+@media (max-width: 736px) {
```

Comment:
https://getbootstrap.com/docs/4.1/content/typography/#responsive-typography

check this out, bootstrap has sm, md, lg breakpoints you can try using